source/sdk/threading/Thread.ooc

```diff
@@ -31,6 +31,24 @@ Thread: abstract class {
 		}
 		null
 	}
+	currentThreadId: static func -> Long {
+		version (unix || apple) {
+			return pthread_self() as Long
+		}
+		version (windows) {
+			return GetCurrentThread() as Long
```

Comment:
I was going to complain but realized the above two methods are like this too. I've made an issue #1261 where I'll try to make files more consistent regarding `return`.